aspect/testing/tests/src/com/google/idea/blaze/aspect/java/pluginprocessorjars/BazelPluginProcessorJarTest.java

```diff
@@ -34,8 +34,8 @@
 public class BazelPluginProcessorJarTest extends BazelIntellijAspectTest {
   // blaze and bazel have different annotation processor paths & names.
   private static final String JAR_STR =
-      jarString("auto-value-1.6.2.jar", /*iJar=*/ null, /*sourceJar=*/ null);
-  private static final String OUTPUT_GROUP_FILES = "../auto_value/auto-value-1.6.2.jar";
+      jarString("external/rules_jvm_external~5.3~maven~maven/com/google/auto/value/auto-value/1.10.2/processed_auto-value-1.10.2.jar", /*iJar=*/ null, /*sourceJar=*/ null);
```

Comment:
I'm not sure if this is the right approach or we should update the aspect to correctly strip the new external dependencies path prefix. we need to look more into why it is needed.